### DISCUSSION TITLE
Sequence/script not loading

### DISCUSSION THREAD

**Discussion by shen-y-eric** [GMOD/jbrowse-components#2076] [Q&A]
Hi, I'm just a beginner following the tutorial for embedding JBrowse 2 into my website. I copy and pasted the code provided in the tutorial, but as you can see in the screenshot below, the sequence selection just loads forever. I suspect it has something with the script not loading properly, but I haven't been able to figure it out. I am currently running this with just <code>npx serve</code>

![image](https://user-images.githubusercontent.com/70082457/123314414-6b6c7e80-d4df-11eb-9c64-8502468f1e7b.png)

**Comment by garrettjstevens** [comment]
I can't say for sure, since the code from the tutorial is working for me, but I would first double-check that the aliases file is correctly formatted and the name of the file is exactly the same as it is specified in assembly.js.
